packages/react/src/feature-link-list/FeatureLinkListItem.tsx

```diff
@@ -1,4 +1,4 @@
-import { ReactNode } from 'react';
+import { ReactNode, useId } from 'react';
```

Comment:
eslint should have stopped you from importing `useId` from react. It needs to be imported from core instead to support older versions of React